tests/test_serving.py

```diff
@@ -114,6 +119,22 @@ def test_reloader_sys_path(tmp_path, dev_server, reloader_type):
     assert client.request().status == 200
 
 
+def test_watchdog_reloader_ignores_opened(tmp_path, dev_server):
+    with patch.object(WatchdogReloaderLoop, "trigger_reload") as mock_trigger_reload:
```

Comment:
I think there's a patch decorator that can be used instead to avoid indenting the entire test.